assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -257,6 +257,12 @@ che.infra.kubernetes.server_strategy=multi-host
 # - 'gateway': Exposes servers using reverse-proxy gateway.
 che.infra.kubernetes.singlehost.workspace.exposure=native
 
+# Defines the way how to expose devfile endpoints, thus end-user's applications, in single-host server strategy.
+# They can either follow the single-host strategy and be exposed on subpaths, or they can be exposed on subdomains.
+# - 'multi-host': expose on subdomains
+# - 'single-host': expose on subpaths
+che.infra.kubernetes.singlehost.workspace.devfile_endpoint_exposure=single-host
```

Comment:
I think we were agreed to set `'multi-host` as default - no?

Reply:
yes